lib/page.go

```diff
@@ -132,16 +136,24 @@ func (page *Page) Is(path string) bool {
 	return page.Url() == path || page.Path == path
 }
 
+//will be used for dinamicly created pages
+func (page *Page) WasRead(set ...bool) bool {
+	if len(set) > 0 {
```

Comment:
if `set` has value, page's unexported  field takes the value. Always return `page.wasread`. Changed in commit.